### PR TITLE
96065 - filter date placeholder fix

### LINKED ISSUE #158: 96065 - [Newsroom] Newsfeed- Editable filter date placeholder
**Severity - 3 - Medium**

**Steps to Reproduce:** 
1. Open URL https://main--accenture-newsroom--hlxsites.hlx.live/
2. Click on Filter place holder- Date Range
**Actual:** Placeholder is editable
image

**Expected:** Placeholder is not editable
image

Azdo link - https://dev.azure.com/accenturecio26/WebContentDigAssetMgmt_PR1093/_workitems/edit/96065

CC: @ravkiran @amol-anand  @raymond-h-dev

### PR DESCRIPTION
**Pull request by raymond-h-dev**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #158 

Test URLs:
- Before: https://main--accenture-newsroom--hlxsites.hlx.page/
- After: https://96065-filter-date-placeholder--accenture-newsroom--hlxsites.hlx.live/

### CODE REVIEW

**Review by ravkiran — CHANGES_REQUESTED** [review]
@raymond-h-dev since the element is input text, even though you put the value as placeholder, user will still be able to edit it 
image

You may have to set read only and update programmatically when the user selects the dates

**Comment by raymond-h-dev** [comment]
> @raymond-h-dev since the element is input text, even though you put the value as placeholder, user will still be able to edit it image
> 
> You may have to set read only and update programmatically when the user selects the dates

@ravkiran  good point, however we just need to follow what's in the live site right now. 

what we can do after we merge the code is ask for business approval from business team for your suggested enhancement, then create/log ticket for this and update this or create a new github issue.

**Comment by raymond-h-dev** [comment]
@zozo-j  let's get business approval for @ravkiran  suggested enhancement

**Review by ravkiran — APPROVED** [review]

**Pull request merged**
